src/main/java/org/jabref/gui/strings/StringViewModel.java

```diff
@@ -0,0 +1,105 @@
+package org.jabref.gui.strings;
+
+import java.util.function.Function;
+import java.util.regex.Pattern;
+
+import javafx.beans.property.ReadOnlyBooleanProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+import org.jabref.logic.l10n.Localization;
+
+import de.saxsys.mvvmfx.utils.validation.CompositeValidator;
+import de.saxsys.mvvmfx.utils.validation.FunctionBasedValidator;
+import de.saxsys.mvvmfx.utils.validation.ValidationMessage;
+import de.saxsys.mvvmfx.utils.validation.ValidationStatus;
+import de.saxsys.mvvmfx.utils.validation.Validator;
+
+public class StringViewModel {
+
+    private final static Pattern IS_NUMBER = Pattern.compile("-?\\d+(\\.\\d+)?");
+
+    private final StringProperty label = new SimpleStringProperty();
+    private final StringProperty content = new SimpleStringProperty();
+
+    private final Validator labelValidator;
+    private final Validator contentValidator;
+    private final CompositeValidator combinedValidator;
+
+    private final Function<String, ValidationMessage> labelValidatorFunction = input -> {
```

Comment:
Any reason why you define this methods as `Function` variables instead of directly as static methods?